src/variables.py

```diff
@@ -78,3 +77,24 @@
 
 # List of ids of staking modules in which the depositor bot will make deposits
 DEPOSIT_MODULES_WHITELIST = [int(module_id) for module_id in os.getenv('DEPOSIT_MODULES_WHITELIST', '1').split(',')]
+
+# All non-private env variables to the logs in main
+PUBLIC_ENV_VARS = {
```

Comment:
Maybe also add PRIVATE_ENV_VARS, to make sure all vars are in PUBLIC_ENV_VARS or PRIVATE_ENV_VARS?

And make sanity check that 
`set(PRIVATE_ENV_VARS.keys()) not in set(PUBLIC_ENV_VARS.keys())`

Reply:
Done